LoopMusic/Source/ViewControllers/MusicPlayerViewController.swift

```diff
@@ -49,6 +49,8 @@ class MusicPlayerViewController: UIViewController, LoopScrubberContainer, UIAdap
            showErrorMessage(error: error)
         }
         
+        updatePlaylistChange()  // Needed if starting from "All Tracks"
```

Comment:
Rather than a separate call here, the notification can be used by setting the current playlist to All tracks when loading the settings if there's no playlist loaded.

Reply:
Resolved in 2b593b3